example/storybook/src/components/Forms/FormControl/index.stories.mdx

```diff
@@ -275,6 +301,607 @@ The following are the lists of all the libraries and packages the component reli
 - `@react-native-aria/focus`
 - `react-native-svg`
 
+### Example
+
+The Examples section provides visual representations of the different variants of the component, allowing you to quickly and easily determine which one best fits your needs. Simply copy the code and integrate it into your project.
+
+#### Form Control with Input
+
+<AppProvider>
+  <CodePreview
+    showComponentRenderer={true}
+    showArgsController={false}
+    metaData={{
+      code: `
+        <FormControl minWidth='$80'>
+          <FormControl.Label mb='$1'>
+            <FormControl.Label.Text color='$text500' lineHeight='$md'>
+              Name
+            </FormControl.Label.Text>
+          </FormControl.Label>
+          <Input>
+            <Input.Input type="text"  py='$2' px='$3' />
+          </Input>
+          <FormControl.Helper mt='$1'>
+            <FormControl.Helper.Text fontSize='$sm' lineHeight='$md'>
+              What would you like people to call you?
+            </FormControl.Helper.Text>
+          </FormControl.Helper>
+        </FormControl>
+      `,
+      transformCode: (code) => {
+        return transformedCode(code);
+      },
+      scope: {
+        Wrapper,
+        FormControl,
+        Icon,
+        Input,
+        WarningIcon,
+      },
+      argsType: {},
+    }}
+  />
+</AppProvider>
+
+#### Form Control with Radio
+
+<AppProvider>
+  <CodePreview
+    showComponentRenderer={true}
+    showArgsController={false}
+    metaData={{
+      code: `
+        <FormControl>
+          <FormControl.Label>
+            <Heading color='$text900' fontSize='$md' lineHeight='$md'>
```

Comment:
Make sure that it is not rendering h1 on web.

Reply:
checked, rendering h4